docs/source/Contribute.md

```diff
@@ -411,6 +411,7 @@ $ docker exec intelowl_uwsgi python3 manage.py test tests.analyzers_manager.test
 
 
 All the frontend tests must be run from the folder `frontend`.
+The tests can contain log messages, you can suppress then with the environment variable `SUPPRESS_JEST_LOG=True`.
```

Comment:
@drosetti `:` instead of `,`. Always ask review of the written documentation if possible

Reply:
It will be fix in the next pr with the update of the dependencies